lib/ruby_lsp/test_reporter.rb

```diff
@@ -67,5 +76,28 @@ def send_message(method_name, params)
         $stdout.write("Content-Length: #{json_message.bytesize}\r\n\r\n#{json_message}")
       end
     end
+
+    class IOWrapper < SimpleDelegator
+      #: (Array[String]) -> void
```

Comment:
Splats shouldn't include the `Array`. You only declare the expected type of the items ([docs](https://sorbet.org/docs/sigs#rest-parameters))
```suggestion
      #: (String) -> void
```